.vscode/settings.json

```
@@ -0,0 +1,5 @@
+{
+    "python.linting.pylintEnabled": false,
```

```suggestion
    "python.linting.pylintEnabled": true,
```
Btw, it's best practice adding a newline at the end of file